qa/L0_model_config/compare_status.py

```diff
@@ -26,49 +26,60 @@
 
 import argparse
 import os
-import os.path
 import sys
-from tensorrtserver.api import *
-import tensorrtserver.api.server_status_pb2 as server_status
-import tensorrtserver.api.model_config_pb2 as model_config
+import json
+import tritonhttpclient.core as httpclient
+import tritongrpcclient.core as grpcclient
+from tritonhttpclient.utils import *
 from google.protobuf import text_format
+from google.protobuf import json_format
+import tritongrpcclient.model_config_pb2 as mc
 
 FLAGS = None
 
-def _get_server_status(url="localhost:8000", protocol=ProtocolType.HTTP, model_name=None):
-   ctx = ServerStatusContext(url, protocol, model_name, False)
-   return ctx.get_server_status()
-
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--expected_dir', type=str, required=True,
-                        help='Directory containing expected output files')
-    parser.add_argument('--model', type=str, required=True,
-                        help='Model name')
-    FLAGS, unparsed = parser.parse_known_args()
-
-    status = _get_server_status(model_name=FLAGS.model)
-    status_config = status.model_status[FLAGS.model].config
+   parser = argparse.ArgumentParser()
+   parser.add_argument('--expected_dir', type=str, required=True,
+                     help='Directory containing expected output files')
+   parser.add_argument('--model', type=str, required=True,
+                     help='Model name')
+   FLAGS, unparsed = parser.parse_known_args()
 
-    nonmatch = list()
+   # TODO Test HTTP as well. Requires additional handling since it is reported as Json. 
```

Comment:
Use FIXME to tag issues. Also instead of this being a loop you probably just want to test them one after the other since as you note the testing code will be different. So you can have the GRPC code be complete and just have a FIXME for needing to add the HTTP part

Reply:
Fixed by #1424. No additional handling is needed anymore